.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,83 @@
+
+In order to ease downstream packaging of Rust binaries, an archive of vendored dependencies is also provided (only relevant for offline builds).
+
+## Requirements
+
+This guide requires:
+
+ * a web browser (and network connectivity)
+ * `git`
+ * GPG setup and personal key for signing
+ * `cargo` (suggested: latest stable toolchain from [rustup][rustup])
+ * `cargo-release` (suggested: `cargo install -f cargo-release`)
+ * A verified account on crates.io
+ * An account on quay.io
+ * Write access to this GitHub project
+ * Upload access to this project on GitHub, crates.io, and quay.io
+
+## Release checklist
+
+These steps show how to release version `x.y.z` on the `origin` remote (this can be checked via `git remote -av`).
+Push access to the upstream repository is required in order to publish the new tag and the PR branch.
+
+- make sure the project is clean and prepare the environment:
+  - [ ] `cargo test`
+  - [ ] `cargo clean`
+  - [ ] `git clean -fd`
+  - [ ] `export RELEASE_VER=x.y.z`
+  - [ ] `export UPSTREAM_REMOTE=origin`
+
+:warning:: `UPSTREAM_REMOTE` should reference the locally configured remote that points to the upstream git repository.
+
+- create release commits on a dedicated branch and tag it:
+  - [ ] `git checkout -b release-${RELEASE_VER}`
+  - [ ] `cargo release` (and confirm the version when prompted)
+
+- open a PR for this release:
+  - [ ] `git push ${UPSTREAM_REMOTE} release-${RELEASE_VER}`
+  - [ ] open a web browser and create a PR for the branch above
+  - [ ] make sure the resulting PR contains exactly two commits
+  - [ ] in the PR body, write a short changelog with relevant changes since last release
+
+- [ ] get the PR reviewed, approved and merged
+
+- publish the artifacts (tag and crate):
+  - [ ] `git push ${UPSTREAM_REMOTE} v${RELEASE_VER}`
+  - [ ] make sure the upstream tag matches the local tag: `git fetch --tags --verbose ${UPSTREAM_REMOTE} 2>&1 | grep ${RELEASE_VER}`
+  - [ ] `git checkout v${RELEASE_VER}`
+  - [ ]  make sure the tag is what you intend to release; if so this will show an empty output: `git diff release-${RELEASE_VER}~1 v${RELEASE_VER}`
+  - [ ] `cargo publish`
+
+- assemble vendor archive:
+  - [ ] `cargo vendor`
+  - [ ] `tar -czf target/coreos-installer-${RELEASE_VER}-vendor.tar.gz vendor`
+
+- publish this release on GitHub:
+  - [ ] open a web browser and [create a GitHub Release](https://github.com/coreos/coreos-installer/releases/new) for the tag above
+  - [ ] write a short changelog (i.e. re-use the PR content) and publish the release
+  - [ ] upload `target/coreos-installer-${RELEASE_VER}-vendor.tar.gz`
+  - record digests of local artifacts:
+    - [ ] `sha256sum target/package/coreos-installer-${RELEASE_VER}.crate`
+    - [ ] `sha256sum target/coreos-installer-${RELEASE_VER}-vendor.tar.gz`
+  - [ ] publish release
+
+- update the `stable` tag on Quay:
```

Comment:
Do you think there some chance of confusion with the `stable` FCOS stream? Should we call this in a way that only covers freshness (e.g. `latest-tag`, `released` or similar)?

Reply:
Good point.  Changed to `release`.